src/arena.rs

```diff
@@ -112,9 +118,18 @@ fn test_common<T: UnsafeArenaWithMembershipCheck<&'static str>>(arena: &mut T) {
 /// `UnsafeArena` implementation that relies on the system memory allocator.
 pub mod sys {
     use super::*;
+    #[cfg(not(feature = "std"))]
+    use core::mem::{transmute, transmute_copy, ManuallyDrop};
+    #[cfg(not(feature = "std"))]
+    use core::ptr::read;
+    #[cfg(not(feature = "std"))]
+    use core::ptr::NonNull;
+
+    #[cfg(feature = "std")]
     use std::mem::{transmute, transmute_copy, ManuallyDrop};
+    #[cfg(feature = "std")]
     use std::ptr::read;
-
+    #[cfg(feature = "std")]
     use std::ptr::NonNull;
```

Comment:
Remove all `#[cfg(feature ="std")] use std::...`.

("Suggested change" didn't work for this one.)